open-banking-accelerator/internal-apis/internal-swaggers/dynamic_client_registration_internal.yaml

```diff
@@ -0,0 +1,379 @@
+swagger: '2.0'
+info:
+  title: Dynamic Client Registration API
+  version: v1.0
+  description: >
+    This specification defines the APIs for a TPP to submit a Software Statement
+    Assertion to an ASPSP for the purpose of creating OAuth clients that are
+    registered with ASPSP.
+  contact:
+    name: Wso2
+    url: 'http://wso2.com/solutions/financial/open-banking/'
+    email: "openbankingdemo@wso2.com"
+  license:
+    name: WSO2 Commercial License
```

Comment:
maybe we need to remove this?

Reply:
fixed with 1d962307c7fd1995b270ac4d410f974b31a84471

Reply:
fixed with 64b4e7069404350d071cdb94c319633f36a3cc11